noir/noir-repo/docs/docs/noir/concepts/functions.md

```diff
@@ -184,14 +184,16 @@ See [Lambdas](./lambdas.md) for more details.
 
 Attributes are metadata that can be applied to a function, using the following syntax: `#[attribute(value)]`.
 
```

Comment:
hmm if we're going to iterate on noir docs from here we should try and set up so these dont trigger e2e